MovieApp/MovieApp/App/Features/MovieBrowser/Components/SubcategoryView/SubcategoryView.swift

```diff
@@ -8,7 +8,7 @@ class SubcategoryView: UIView {
     let maxNumberOfSubcategoryItems: Int = 8
     
     var selectedSubcategory: Int!
-    var subcategories: [LocalSubcategory]?
+    var subcategories: [Genre]?
```

Comment:
Shouldn't this be named `genres`?
And everywhere else.

Reply:
Renamed everything with "subcategory" in it's name to "genre".
[Here](https://github.com/lihter/Movie-App/pull/36/commits/0c67490efb03a8db70a0474781bcd26cbcc0ae70#diff-5e12aa457f4430537ff0302ead016812ca42213efc1a47ddf8d5cb44151300a0R10)